recipes/ncbi-amr/compile-fix.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/Makefile b/Makefile
+index 1417b54..99a24a4 100644
+--- a/Makefile
++++ b/Makefile
+@@ -27,7 +27,8 @@ CXX=g++
+ COMPILE.cpp= $(CXX) $(CPPFLAGS)  -c
+ 
+ #prefix = /panfs/pan1.be-md.ncbi.nlm.nih.gov/bacterial_pathogens/backup
+-prefix=/usr/local
++#prefix=/usr/local
++prefix=${PREFIX}
```

Comment:
You can just write
```
make prefix=$PREFIX
```
to achieve the same. Variables passed like that always override everything in the Makefile.

Reply:
Will do, thanks.